crates/holochain_persistence_lmdb/src/cas/lmdb.rs

```diff
@@ -33,55 +31,109 @@ impl Debug for LmdbStorage {
 }
 
 impl LmdbStorage {
-    pub fn new<P: AsRef<Path> + Clone>(
-        db_path: P,
-        initial_map_bytes: Option<usize>,
-    ) -> LmdbStorage {
-        LmdbStorage {
+    pub fn new<P: AsRef<Path> + Clone>(db_path: P, initial_map_bytes: Option<usize>) -> Self {
```

Comment:
```suggestion
    pub fn new<P: AsRef<Path> + Clone>(db_path: P, initial_map_size: Option<usize>) -> Self {
```
I instantly read "bytes" to mean a collection of bytes and was surprised that LMDB just lets you create databases out of arbitrary bytes. Size is a better name in my view